lib/modules/manager/dockerfile/extract.ts

```diff
@@ -176,14 +177,19 @@ export function getDep(
   // Resolve registry aliases first so that we don't need special casing later on:
   for (const [name, value] of Object.entries(registryAliases ?? {})) {
     const escapedName = escapeRegExp(name);
-    const groups = regEx(`(?<prefix>${escapedName})/(?<depName>.+)`).exec(
+    const groups = regEx(`${escapedName}/(?<depName>.+)`).exec(
```

Comment:
technically we can use a registy alias like this: `domain.test=>docker.io`

this will replace like this `some.domain.test/some/image` to `docker.io/some/image`

I'm not sure if this is intended behavior 🤔